transactions/main.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+// Episode represents the schema for the "Episodes" collection
+type Episode struct {
+	ID          primitive.ObjectID `bson:"_id,omitempty"`
+	Podcast     primitive.ObjectID `bson:"podcast,omitempty"`
+	Title       string             `bson:"title,omitempty"`
+	Description string             `bson:"description,omitempty"`
+	Duration    int32              `bson:"duration,omitempty"`
+}
+
+func main() {
+	client, err := mongo.Connect(context.TODO(), options.Client().ApplyURI(os.Getenv("ATLAS_URI")))
+	if err != nil {
+		panic(err)
+	}
+	defer client.Disconnect(context.TODO())
+
+	database := client.Database("quickstart")
+	episodesCollection := database.Collection("episodes")
+
+	session, err := client.StartSession()
```

Comment:
Add a `defer session.EndSession(ctx)` call after the error check instead of ending it after the `WithSession` returns so it's always cleaned up

Reply:
This change will happen in the next commit.